pilot/server/llmserver.py

```diff
@@ -103,7 +103,8 @@ def generate_stream_gate(self, params):
                 else:
                     print("output: ", output)
                 # return some model context to dgt-server
-                ret = {"text": output, "error_code": 0, "model_context": model_context}
+                ret = {"text": output, "error_code": 0, "model_context": model_context,
+                       "relations": params.get("relations")}
```

Comment:
The `"relations"` just used by `ChatKnowledge`,  There is no need to support it in a common code, also not in `BaseOutputParser`.